### ISSUE TITLE
Stacked Hourglass results only 65mm

### ISSUE THREAD

**Issue by Yiran-Zhu** [garyzhao/SemGCN#35]
I downloaded the Github repo and obtained the SH_2d features according to the instructions, but I directly used the warehouse, and the results were not as good as reported, only about 65. I don't know why.

**Comment by garyzhao** [comment]
Hi @yiranupup 

I guess you might use the last model for evaluation.

The best model is saved with the suffix of "best". Please use this model for evaluation.

And our results are reported by using the evaluation model which computes an average sore over per action, so will be slightly better than the results during training.

Best,
Long

**Comment by yiranupup** [comment]
Thanks for your reply. Considering the time, I use batchsize of 256. Is that the reason? Does batchsize difference make a big difference in the results?

**Comment by garyzhao** [comment]
> Thanks for your reply. Considering the time, I use batchsize of 256. Is that the reason? Does batchsize difference make a big difference in the results?

I guess using batch size of 256 should be fine, but you might need to train the model with more epochs, e.g., double or quadruple the total number of epochs. This is because the total number of iteration is reduced if you change the batch size to 256. Or you can try to choose a larger initial learning rate instead.

Best,
Long

**Comment by yiranupup** [comment]
I see. Thank you very much for your clarification.
